src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -508,10 +509,10 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'type': 'string',
 				'default': (() => {
 					if (isMacintosh && isNative) {
-						return '${activeEditorShort}${separator}${rootName}'; // macOS has native dirty indicator
+						return '${activeEditorShort}${separator}${rootName}${separator}${profileName}'; // macOS has native dirty indicator
 					}
 
-					const base = '${dirty}${activeEditorShort}${separator}${rootName}${separator}${appName}';
+					const base = '${dirty}${activeEditorShort}${separator}${rootName}${separator}${appName}${separator}${profileName}';
```

Comment:
Why would the profile be after the app name?

Reply:
Good catch!! I just copy pasted the format from MacOS section. Fixed.